x-pack/plugins/enterprise_search/public/applications/analytics/api/index/fetch_analytics_collections_api_logic.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AnalyticsCollection } from '../../../../../common/types/analytics';
+
+import { createApiLogic } from '../../../shared/api_logic/create_api_logic';
+import { HttpLogic } from '../../../shared/http';
+
+export const fetchAnalyticsCollections = async () => {
+  const { http } = HttpLogic.values;
+  const route = '/internal/enterprise_search/analytics/collections';
+  const response = await http.get<AnalyticsCollection[]>(route);
+
+  return response;
+};
+
+export const fetchAnalyticsCollectionsAPILogic = createApiLogic(
```

Comment:
This is a nit, but for consistency I'll add it:

```suggestion
export const FetchAnalyticsCollectionsAPILogic = createApiLogic(
```

Reply:
You'll need to update it everywhere it's referenced as well, obviously.